x-pack/plugins/alerting/server/routes/lib/rewrite_rule.ts

```diff
@@ -63,7 +63,9 @@ export const rewriteRule = ({
     connector_type_id: actionTypeId,
     ...(frequency
       ? {
-          frequency,
+          summary: frequency.summary,
+          notify_when: frequency.notifyWhen,
+          throttle: frequency.throttle,
```

Comment:
`{
          ...frequency,
          notify_when: frequency.notifyWhen,
 }`

Reply:
We need to omit `notifyWhen` and replace it with `notify_when`. I could shorten this by importing lodash `omit` and doing `{ ...omit(frequency, 'notifyWhen'), notify_when: frequency.notifyWhen }` but with only three props to pass in it feels like a lot to add a whole new `import` for. What do you think?

Reply:
Yes, I get it now. Yes, probably it a lot. But will be more obvious for someone who does not know context. So do as you think right.